### DISCUSSION TITLE
Recommend

### DISCUSSION THREAD

**Discussion by Andriusito** [jebbyk/OSBES-minecraft-bedrock-edition-shader#139] [General]
I would like this shader to be optimized more, since its previous version works more fluid for me than the current one, currently the latest version drops a few fps, I would also like to see the bright ores implemented

**Comment by OpenSauce04** [comment]
Due to the shader being unfinished, performance may fluctuate slightly between versions. These drops in performance will likely be ironed out at a later date.
We are asked frequently about bright ores, but we have already decided that we will not be implementing them, sorry!

**Reply by Andriusito** [reply]
Don't worry, it's still a great shader, by the way, will it ever be compatible with textures?
